packages/perseus/src/util/parse-perseus-json/perseus-parsers/versioned-widget-options.test.ts

```diff
@@ -114,9 +118,9 @@ describe("versionedWidgetOptions parser", () => {
     });
 
     it("migrates through intermediate versions", () => {
-        const parser = versionedWidgetOptions(parseOptionsV2)
-            .withMigrationFrom(parseOptionsV1, migrateV1ToV2)
-            .withMigrationFrom(parseOptionsV0, migrateV0ToV1).parser;
+        const parser = versionedWidgetOptions(2, parseOptionsV2)
+            .withMigrationFrom(1, parseOptionsV1, migrateV1ToV2)
+            .withMigrationFrom(0, parseOptionsV0, migrateV0ToV1).parser;
```

Comment:
This reads more clearly. Thanks!  I like that the major version is mentioned explicitly here.

Reply:
Looks like I forgot to change the base branch... this change belongs to another PR. Sorry about that!